mautic/mautic
# [UX] Change Alias tooltips when creating Segments, Assets, Landing pages, Forms, from "Alphanumeric string" to "Letters and numbers"

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
The current tooltip for the Alias field in the creation of Segments, Assets, Landing pages, and Forms, which is “Alphanumeric string”, presents an opportunity for enhancement. It would be beneficial to use a more commonly understood term. The proposed solution in this pull request is to change the tooltip to “Letters and numbers”, which is a more straightforward description of the input requirements.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Segments
3. Create a new segment
4. Hover your mouse above the Alias field
5. Read the tooltip

![image](https://github.com/mautic/mautic/assets/116097999/a7d86501-083c-4da5-8b64-7940fb7d2510)
